src/pages/search/index.tsx

```diff
@@ -3,17 +3,22 @@ import { GetStaticPropsContext } from "next";
 import { useRouter } from "next/router";
 import { gql, useQuery } from "@apollo/client";
 import { Koros } from "hds-react";
+import dynamic from "next/dynamic";
 
 import { Connection, Item, Keyword, SearchResult } from "../../types";
 import searchApolloClient from "../../client/searchApolloClient";
 import { getNodes } from "../../client/utils";
-import initializeCmsApollo from "../../client/cmsApolloClient";
-import SearchPageSearchForm from "../../components/search/searchPageSearchForm/SearchPageSearchForm";
 import Page from "../../components/page/Page";
 import Section from "../../components/section/Section";
 import SearchResultCard from "../../components/card/searchResultCard";
 import SearchList from "../../components/list/SearchList";
 import styles from "./search.module.scss";
+import initializeCmsApollo from "../../client/cmsApolloClient";
+import SearchHeader, {
+  ShowMode,
+} from "../../components/search/searchHeader/SearchHeader";
+import getURLSearchParamsFromAsPath from "../../util/getURLSearchParamsFromAsPath";
+import getShowMode from "../../util/getShowMode";
```

Comment:
Imports are reordered in an odd way to my eye.

Reply:
Same thing here, these got messed up while I was resolving merge conflicts. I'll fix these as well.